source/localizable/ember-inspector/index.md

```diff
@@ -0,0 +1,7 @@
+The Ember Inspector is a browser add-on designed to help you understand and debug your Ember applications. You can install it on [Google Chrome](installation/#toc_google-chrome), [Firefox](installation/#toc_firefox) and [other browsers](http://localhost:4567/ember-inspector/installation/#toc_via-bookmarklet) (via a bookmarklet)
+
+Here's a brief video showcasing some of the features of the Inspector:
+
+<div class="video">
+  <iframe width="640" height="360" src="https://www.youtube.com/embed/6Yl1j2thKac?feature=player_detailpage" frameborder="0" allowfullscreen></iframe>
```

Comment:
Thanks, I've updated it to be responsive using [this technique](https://www.h3xed.com/web-development/how-to-make-a-responsive-100-width-youtube-iframe-embed):

<img width="1035" alt="screen shot 2016-09-30 at 09 08 09" src="https://cloud.githubusercontent.com/assets/2526/18984970/8b409d44-86ed-11e6-8416-9a175a41372e.png">

<img width="569" alt="screen shot 2016-09-30 at 09 08 27" src="https://cloud.githubusercontent.com/assets/2526/18984981/9661f286-86ed-11e6-9dba-6ca7f9f0a137.png">